### PR TITLE
Fix DHCP example for cobbler autoinst and add per arch naming

### PR DESCRIPTION
**Pull request by aaannz**
# Description

* Suggest correct GRUB binary filename in DHCPd example for autoinstalling clients.
* Add table mapping for different architectures and grub filenames. Source of the data is [cobbler configuration](https://github.com/openSUSE/cobbler/blob/uyuni/master/config/cobbler/settings.yaml#L49)
* Add note that cobbler managed dhcp is not supported

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [X] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes https://github.com/SUSE/spacewalk/issues/22243

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Fix DHCP example for cobbler autoinst and add per arch naming
```
Comment (keichwa) [inline .changelog:7]:
```suggestion
- Fixed DHCP example for Cobbler autoinstallation and added one per
  architecture in Client Configuration Guide
```

**Review by keichwa — APPROVED** [review]
suggestion to normalize the changelog entry.

File: .changelog [hunk 2]
```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Fixed DHCP example for Cobbler autoinstallation and added one per
+  architecture in Client Configuration Guide
```
Comment (keichwa) [inline .changelog:8]:
```suggestion
  architecture in Client Configuration Guide (bsc#1214041)
```

**Comment by keichwa** [comment]
@aaannz If I understand it correctly it needs to be ported to `master`.  I can prepare the port.

**Comment by aaannz** [comment]
> @aaannz If I understand it correctly it needs to be ported to `master`. I can prepare the port.

Yes, please.

**Pull request merged**